tests/test_on_data/ophys/test_imaging_interfaces.py

```diff
@@ -41,6 +43,15 @@
 except ImportError:
     from setup_paths import OPHYS_DATA_PATH, OUTPUT_PATH
 
+skip_on_darwin_arm64 = pytest.mark.skipif(
+    platform.system() == "Darwin" and platform.machine() == "arm64",
+    reason="Tests are skipped on macOS ARM64 due to platform limitations.",
+)
+skip_if_isx_not_installed = pytest.mark.skipif(
```

Comment:
It is OK if the tests fail because we don't the import. We should check for that.